src/rpc/ngHandlers/AccountInfo.h

```diff
@@ -0,0 +1,112 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2023, the clio developers.
+
+    Permission to use, copy, modify, and distribute this software for any
+    purpose with or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL,  DIRECT,  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#pragma once
+
+#include <backend/BackendInterface.h>
+#include <rpc/RPCHelpers.h>
+#include <rpc/common/Types.h>
+#include <rpc/common/Validators.h>
+
+#include <boost/asio/spawn.hpp>
+
+namespace RPCng {
+class AccountInfoHandler
+{
+    std::shared_ptr<BackendInterface> sharedPtrBackend_;
+
+public:
+    struct Output
+    {
+        uint32_t ledgerIndex;
+        std::string ledgerHash;
+        ripple::STLedgerEntry accountData;
+        std::optional<std::vector<ripple::STLedgerEntry>> signerLists;
+        // validated should be sent via framework
+        bool validated = true;
+
+        Output(
+            uint32_t ledgerId,
+            std::string ledgerHash,
+            ripple::STLedgerEntry sle,
+            std::vector<ripple::STLedgerEntry> signerLists)
+            : ledgerIndex(ledgerId)
+            , ledgerHash(std::move(ledgerHash))
+            , accountData(std::move(sle))
+            , signerLists(std::move(signerLists))
+        {
+        }
+
+        Output(
+            uint32_t ledgerId,
+            std::string ledgerHash,
+            ripple::STLedgerEntry sle)
+            : ledgerIndex(ledgerId)
+            , ledgerHash(std::move(ledgerHash))
+            , accountData(std::move(sle))
+        {
+        }
+    };
+
+    // "queue" is not available in Reporting mode
+    // TODO: ident is not in document
```

Comment:
Consider adding an issue for the doc update and posting a link to it here in this todo 🙇